ibis/expr/operations.py

```diff
@@ -516,6 +523,65 @@ class Log10(Logarithm):
     """Logarithm base 10"""
 
 
+class Degrees(UnaryOp):
+    """Converts radians to degrees"""
+    arg = Arg(rlz.floating)
+    output_type = rlz.shape_like('arg', dt.float)
+
+
+class Radians(UnaryOp):
+    """Converts radians to degrees"""
+    arg = Arg(rlz.floating)
+    output_type = rlz.shape_like('arg', dt.float)
```

Comment:
These are definitely float32 and not double?

Reply:
The output type should be the same as the input type, so if it's a `float32` return a `float32` otherwise return a `float64`.

Reply:
ok .. I changed to 

```
output_type = rlz.shape_like('arg', dtype=arg.type())
```

Reply:
sorry, I changed to:

```
    def output_type(self):
        return rlz.shape_like('arg', dtype=self.arg.type())
```

Reply:
Actually .. I think the input could be defined as numeric

> The output type should be the same as the input type, so if it's a float32 return a float32 otherwise return a float64.

could you point me some example to do that?

or could we just put as float64 and treat different output (if necessary) in a follow up PR?